public/bjerk-favicon.svg

```diff
@@ -0,0 +1,21 @@
+<svg width="1119" height="1119" viewBox="0 0 1119 1119" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
SVG favicon? Does that work?

Maybe we should run Prettier on this?

Reply:
We haven't ran Prettier on any of the SVG's, but yeah that might be an idea